netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[redirects]]
+  from = "/*"
+  to = "/index.html"
+  status = 200
```

Comment:
@miguelmota What is this file for?

Reply:
It's a [netlify](https://netlify.com/) redirects config to handle HTML pushState routes since create-react-app is a single page application (all routes are handled in root index.html). 

Without the redirects config the pages will 404 if user refreshes the page. 

The demo link is hosted on netlify so it needed this file. Happy to remove it for this PR (although netlify is a very popular static hosting site)

Reply:
Could you remove it in the PR?

Reply:
Done